templates/webserver/webserver-deployment.yaml

```diff
@@ -65,7 +65,11 @@ spec:
           {{- include "standard_airflow_environment" . | indent 10 }}
       containers:
         - name: webserver
+          {{- if semverCompare ">=2.0.0" .Values.airflowVersion }}
+          image: {{ template "default_airflow_image" . }}
```

Comment:
This entirely removes the ability to use plugins in the webserver.

I feel this needs to be opt-in per deployment somehow.

cc @schnie

Reply:
Is the goal to stop rebooting, or to stop down time of webserver on deploy?

If the later we could change the replacement policy on the deployment

Reply:
good catch, do you mean change pullPolicy for image?

Reply:
The ultimate goal is to prevent seeing the rocketship after they do a code-push. People don't like the webserver downtime. Forgot about plugins/blueprints/custom views though. Dang, that makes it tough.

Reply:
Could we change the default to not restart and allow the user to override when needed?

Reply:
There may be a better UX, but off the top of my head maybe it could be a checkbox on the webserver settings to "Allow Plugins", or something. Under the hood we use that flag to determine if we use `default_airflow_image` or `airflow_image`. 

The caveat of "Allowing Plugins" would be that we keep the current behavior and reboot the webserver.

I _feel_ like a small amount of customers user plugins, so they'd be the exception here, but I really don't know for sure. @paolaperaza @ryw @vparekh94 @petedejoy any idea on this?

Reply:
(FYI - summarized the convo/questions that have come up on this in the GH issue itself: https://github.com/astronomer/issues/issues/2123#issuecomment-730033302)